Neos.Utility.ObjectHandling/Classes/TypeHandling.php

```diff
@@ -114,6 +114,17 @@ public static function isSimpleType(string $type): bool
         return in_array(self::normalizeType($type), ['array', 'string', 'float', 'integer', 'boolean'], true);
     }
 
+    /**
+     * Returns true if the $type is a literal type.
+     *
+     * @param string $type
+     * @return boolean
+     */
+    public static function isLiteralType(string $type): bool
+    {
+        return in_array(self::normalizeType($type), ['false', 'true'], true);
```

Comment:
```suggestion
        return in_array(self::normalizeType($type), ['false'], true);
```
I think `true` is not a valid literal type, only `false` is allowed. `null` would be another value in the same category (but we could discuss that in another PR). https://php.watch/versions/8.2/null-false-types

Reply:
true is valid since php 8.2 that is why i included it. See https://www.php.net/releases/8.2/en.php#null_false_true_types

Reply:
Added `null` to the simpleTypes as it is documented here https://www.php.net/manual/de/language.types.type-system.php

Reply:
Also: I wonder wether we should simply add `true` and `false` to the list of simple types and avoid the seperate "isLiteralType" method. Not sure.